using-the-website/bonds.md

```diff
@@ -1,30 +1,61 @@
 
 ## How to Bond
 
-First, go to Sushiswap and add liquidity. If you do not know how to do so, there is a good guide here. Next, come to our [website](http://olympusdao.finance/#/bond). Make sure "Bond" is selected. Type in the amount of SLP you would like to bond, then click Approve and sign the transaction. Next, click Bond and sign that transaction. **Please note that bonding takes two transactions; approve and bond.**
+#### LP Bond
+
+1. Go to Sushiswap and add liquidity. If you do not know how to do so, there is a good guide on [SushiSwap documentation](https://help.sushidocs.com/guides/how-to-add-tokens-to-sushiswap-exchange-as-an-lp).
+
+2. Head to [Olympus website](http://olympusdao.finance/#/bond). There are two tabs: "Bond" and "Redeem"; make sure "Bond" is selected.
+
+3. Type in the amount of SLP you would like to bond, then click "Approve" and sign the transaction.
+
+4. After the "Approve" transaction has been processed successfully, click "Bond" and sign the transaction. Voila, you have purchased your first SLP bond!
+
+#### DAI Bond
+
+1. Head to [Olympus website](http://olympusdao.finance/#/bondai). There are two tabs: "Bond" and "Redeem"; make sure "Bond" is selected.
+
+2. Type in the amount of DAI you would like to bond, then click "Approve" and sign the transaction.
+
+3. After the "Approve" transaction has been processed successfully, click "Bond" and sign the transaction. Voila, you have purchased your first DAI bond!
+
+*Note: The "Approve" transaction is only needed when bonding for the first time; subsequent bonding only requires you to perform the "Bond" transaction.*
 
 ## **How to Redeem**
 
-Come to our [website](http://olympusdao.finance/#/bond) and select the "Redeem" tab at the top of the window. Click "Redeem" to claim all of your available rewards.
+On the same webpage that you used to purchase bond, select the "Redeem" tab. Click "Redeem" to claim all of your available rewards.
 
 ## Reading the Info
 
 ![](../.gitbook/assets/using-the-website/bonds/modal.png)
 
 **Balance** is your balance of SLP tokens. This is the asset used to create a bond.
 
-**Bond Price** is the price of OHM through bonds. You can calculate the bond price as:
-\(Value of your SLP token / OHM you'll get from the bond\).
+**Bond Price** is the price of OHM you get from bonding. You can calculate the bond price using the following formulae:
+
+- SLP Bond: \(Value of your SLP token / OHM you'll get from bonding\)
+- DAI Bond: \(Value of your DAI token / OHM you'll get from bonding\)
 
 **Market Price** is the market price of OHM.
 
-**You Will Get** tells you how many OHM you will get from bonding the chosen amount of SLP.
+**You Will Get** tells you how many OHM you will get from bonding the chosen amount of SLP or DAI tokens.
```

Comment:
The addition after "you will get from bonding" is somehow confusing. I am not sure what the rest tries to tell me. What about just making it short? 

```suggestion
**You Will Get** tells you how many OHM you will get from bonding.
```